.travis.yml

```diff
@@ -59,18 +59,20 @@ matrix:
     - (test -x $HOME/.cargo/bin/mdbook || cargo install --vers "^0.1" mdbook)
     - cargo install-update -a
     script:
-    - cd docs && mdbook build
+    - (cd docs && mdbook build)
```

Comment:
what do the parans do here? am curious

Reply:
Ah this is because travis generates a literal shell script and without the parens the `cd` changes the directory for the rest of the script, so by using parens here the cwd is changed for just this one step